### PR TITLE
Input

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: styles/component-style/input.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.input-container,input {
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+    background-color: var(--background-color);
+    padding:0.5rem;
```
Comment (Omkar-Ghate) [inline styles/component-style/input.css:6]: can use px here

**Review by Omkar-Ghate — APPROVED** [review]
Try to be consistent with the space between the CSS property name and its value. Rest everything looks great.

File: docs/input.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (dev-enforced) [inline docs/input.html:7]: Change it to Inputs if it contains input related contents.

File: docs/input.html [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    <link rel="stylesheet" href="/styles/main-style.css">
```
Comment (dev-enforced) [inline docs/input.html:8]: Do keep your original stylesheet for this file at the end of the head to make sure that they get applied as per your needs

File: styles/component-style/input.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+.name-input-validation {
+    color:green;
+}
+
+.password-input-validation {
+   color:red;
+}
+
+.input-btn-wrapper,.input-btn {
```
Comment (dev-enforced) [inline styles/component-style/input.css:34]: You can use a utility class for this named as bricks-flex and pass it to the HTML file if you want to apply display:flex to that element

**Review by dev-enforced — APPROVED** [review]
The code looks good to me You can merge it @parulgupta2001 .Suggested some minor changes after which you can merge Do deploy it on Netlify or Vercel so that we can see the live preview of it

**Pull request merged**